### ISSUE TITLE
Missing types for PostView.Record in @atproto/api

### ISSUE THREAD

**Issue by sameoldlab** [bluesky-social/atproto#946]
Interface for `PostView` has an empty object as the properties on `record`. 

Closest type I can find is this?  https://github.com/bluesky-social/atproto/blob/0b70f50eb44bf6a936115f0e24690a42585b69a6/packages/api/src/client/types/app/bsky/feed/post.ts#L15-L29
but seems it's for creating records not viewing them

**Comment by tom-sherman** [comment]
I'm struggling with this also.

Not sure if this is an intentional omission or a bug though.

**Comment by dholms** [comment]
Yes that should probably be an open union with a ref to `app.bsky.feed.post`

See my comment here as well: https://github.com/bluesky-social/atproto/issues/999#issuecomment-1577769209
